src/client/src/containers/ProjectNameAndOutput/index.tsx

```diff
@@ -60,13 +60,22 @@ const messages = defineMessages({
     id: "projectName.ariaProjectName",
     defaultMessage: "Project Name Input"
   },
+  nameTooLong: {
+    id: "projectNameError.nameTooLong",
+    defaultMessage:
+      "Project name can only be " +
+      PROJECT_NAME_CHARACTER_LIMIT +
+      " characters long"
```

Comment:
I know you didn't write this but we should change this to "Project name can only be {characterLimit} characters long" and pass ` PROJECT_NAME_CHARACTER_LIMIT` when you format the message (in this PR or another one)

Reply:
fixed it